Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Marging and padding</title>
  <style>
    body {
      font-family: Arial, sans-serif;
      background-color: #f4f4f4;
      margin: 0;
      padding: 0;
    }

    .content-box {
      margin: 20px;       
      padding: 15px;      
      background-color: #e0e0e0;
      border: 1px solid #ccc;
      border-radius: 5px;
    }

    section {
      width: 600px;       
      height: 400px;      
      background-color: #90caf9; 
      margin: 30px auto; 
      padding: 20px;
      box-sizing: border-box; 
    }

    section h2 {
      font-size: 24px;
      color: #1e88e5;
    }

    section p {
      font-size: 18px;
      color: #333;
    }

  </style>
</head>
<body>

  <div class="content-box">
    <h2>Div with Margin and Padding</h2>
  </div>

  <section>
    <h2>Section with Defined Size</h2>
  </section>

</body>
</html>